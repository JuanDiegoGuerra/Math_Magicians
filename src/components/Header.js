import { NavLink } from 'react-router-dom';
import styles from '../styles/Header.module.css';

const Header = () => (
  <header className={styles.header}>
    <h1>MathMagicians</h1>
    <ul className={styles.nav}>
      <li className={styles.listitem}>
        <NavLink className={styles.link} to="/">
          Home
        </NavLink>
      </li>
      <li className={styles.listitem}>
        <NavLink
          className={styles.link}
          to="/calculator"
        >
          Calculator
        </NavLink>
      </li>
      <li className={styles.listitem}>
        <NavLink
          className={styles.link}
          to="/quote"
        >
          Quote
        </NavLink>
      </li>
    </ul>
  </header>
);
export default Header;
