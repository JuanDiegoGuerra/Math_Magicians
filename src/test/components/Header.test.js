import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../../components/Header';

describe('Header', () => {
  test('Renders the header', () => {
    const { container } = render(
      <Router>
        <Header />
      </Router>,

    );
    expect(container).toMatchSnapshot();
  });
});