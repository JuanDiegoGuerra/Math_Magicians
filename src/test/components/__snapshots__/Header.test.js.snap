// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders the header 1`] = `
<div>
  <header
    class="header"
  >
    <h1>
      MathMagicians
    </h1>
    <ul
      class="nav"
    >
      <li
        class="listitem"
      >
        <a
          aria-current="page"
          class="link active"
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class="listitem"
      >
        <a
          class="link"
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li
        class="listitem"
      >
        <a
          class="link"
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </header>
</div>
`;
